### PR TITLE
Remove dependency on spf13/viper and m4rw3r/uuid

### PR DESCRIPTION
**Pull request by alde**
Build a smaller custom config loading
Use a pre-existing dependency for uuid

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -1,9 +1,105 @@
...
+	// Logging
+	LogLevel  string `yaml:"loglevel"`
+	LogFormat string `yaml:"logformat"`
+
+	// Server
+	Address string `yaml:"address"`
+	Port    int    `yaml:"port"`
+
+	// Database
+	Database       database.TaskDB `yaml:"-" ignored:"true"`
```
Comment (keis) [inline config/config.go]: What is this even doing here?

Comment (alde) [inline config/config.go]: That's one of the three original values on the Config struct

Comment (keis) [inline config/config.go]: Was the purpose of the original struct and is it anything like the new?

Comment (alde) [inline config/config.go]: The purpose of the original was to pass around the version and the database pointer

I think the passing of the database pointer should be changed, but I didn't feel it was in scope of this branch

Comment (keis) [inline config/config.go]: Sure thing but could you at least keep them as distinct types. Config/OtherConfig?

Comment (alde) [inline config/config.go]: if that would make you happy ;)

Comment (keis) [inline config/config.go]: it's not pretty but it's honest :ghost:

Comment (alde) [inline config/config.go]: fixed it to explicitly pass around the database handler where it's needed instead of hiding it on the config object

**Comment by alde** [comment]
And apparently somehow I got a conflict

**Comment by keis** [comment]
:+1: LGTM

**Pull request merged**